src/js/App/Sidenav/index.js

```diff
@@ -9,7 +9,7 @@ const documentationLink = {
     rhel: 'https://access.redhat.com/documentation/en-us/cloud_management_services_for_red_hat_enterprise_linux/1.0/',
     insights: 'https://access.redhat.com/documentation/en-us/red_hat_insights/',
     openshift: 'https://docs.openshift.com/container-platform/4.2/',
-    subscriptions: 'https://access.redhat.com/documentation/en-us/red_hat_insights/1.0/'
+    subscriptions: 'https://access.redhat.com/documentation/Subscription_Central'
```

Comment:
This URL throws 404 is this correct URL?

Reply:
Indeed @karelhala the docs are currently unavailable from my perspective that's behaving correctly. However, to confirm I'll defer towards 
@jeperry @bclarhk @jlprevatt @rblackbu 
to confirm.

Reply:
After some back and forth, looks like we do need to update the PR... the trailing `/` ref needs to be added or it'll continue to `404`. Updating accordingly